libr/core/meson.build

```diff
@@ -97,7 +97,7 @@ r_core = library('r_core', files,
     r_bp,
     libr_shlr
   ],
-  dependencies: [platform_deps],
+  dependencies: [platform_deps] + lmagics,
```

Comment:
Why a `+`, can't you put it in the array?

Reply:
Again, `lmagics` is already an array, so I just appended it to `[platform_deps]`. Does it work even if i put it inside the array?